modules/introduction/partials/new-features-76.adoc

```diff
@@ -86,3 +86,8 @@ See xref:learn:data/expiration.adoc[] for more information.
 
 * You can no longer set the `sendStats` to `false` in Couchbase Server Community Edition clusters.  
 You can still set `sendStats` to `false` on Couchbase Server Enterprise Edition clusters.
+
+* Couchbase Server's LDAP support now has a setting that turns on and off middlebox compatibility.
```

Comment:
> Couchbase Server's LDAP support now has a setting that turns on and off middlebox compatibility.

Couchbase Server's LDAP support now has a setting that turns on and off TLS middlebox compatibility.

Reply:
Updated.